single-node/user-data

```diff
@@ -31,11 +31,19 @@ export K8S_SERVICE_IP=10.3.0.1
 export DNS_SERVICE_IP=10.3.0.10
 
 # Whether to use Calico for Kubernetes network policy.
-export USE_CALICO=false
+export USE_CALICO=true
```

Comment:
yep sorry about that it should have been set to false, must have missed it, amended the last commit.